cashweb-payload/src/payload.rs

```diff
@@ -274,6 +276,28 @@ impl<T: prost::Message + Default> SignedPayload<T> {
     pub fn txs(&self) -> &[BurnTx] {
         &self.burn_txs
     }
+
+    /// Add additional burn transactions to this signed payload to indicate
+    /// additional burns that were added later by others.
+    pub fn add_burn_txs<'a>(&mut self, burn_txs: &'a [BurnTx]) -> Vec<&'a BurnTx> {
+        let mut txid_set = HashSet::<Vec<u8>>::new();
```

Comment:
```suggestion
        let mut txid_set = HashSet::<Sha256d>::new();
```

you should simply use the Sha256d object, it also implements Hash and you get more type safetly (e.g. avoid accidentally adding the serialized tx vec)